dotcms-integration/src/test/java/com/dotcms/http/CircuitBreakerUrlTest.java

```diff
@@ -423,4 +440,34 @@ public void testBadRequest_dontThrow() throws Exception {
         assert (cburl.response() >= 400 && cburl.response() <= 499);
     }
 
+    /**
+     * Given an int status code
+     * Then evaluate it does have a SUCCESS http status
+     */
+    @Test
+    public void test_isSuccessStatusCode() {
```

Comment:
Why are we changing this test class?

Reply:
CircuitBreakerURL logic was included in previous Analytics feature. I need it to relocate it to CircuitBreakerURL in order to be reusable by other classes that need to send HTTP requests. With that change I added an integration test for it.